core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMaterializedViewDefinition.java

```diff
@@ -146,11 +147,13 @@ public int hashCode()
     {
         private final String name;
         private final TypeId type;
+        private final Optional<String> comment;
```

Comment:
Added `comment` field as well in the `ConnectorMaterializedViewDefinition.Column` although `CommentTask` can't set it.
I'm actually not sure why `ConnectorMaterializedViewDefinition` has support for `comment` when there can't be set comments on MVs. BTW why can't we set comments on MVs?

Reply:
> Added comment field as well in the ConnectorMaterializedViewDefinition.Column although CommentTask can't set it.

I don't think we want to add the field in this PR. 

> I'm actually not sure why ConnectorMaterializedViewDefinition has support for comment when there can't be set comments on MVs. BTW why can't we set comments on MVs?

@raunaqmorarka may know it.

Reply:
Comment can be set as part of CREATE MV command. That is tested in `BaseConnectorTest#testMaterializedView`. COMMENT ON MV command isn't implemented because that requires a review about whether separate syntax for commenting on MVs is ANSI sql complaint.
Any changes to MV should be in a separate PR, it would be better to restrict this one to just VIEWs.

Reply:
@findinpath Please let me know if you intentionally left this field. My opinion is it's not necessary.

Reply:
Removed.